nuclear/cmake/Modules/NUClearCompilerSettings.cmake

```diff
@@ -19,6 +19,9 @@ set(CMAKE_MACOSX_RPATH ON)
 # Add some useful places to the RPATH These will allow the binary to run from the build folder
 set(CMAKE_INSTALL_RPATH ${CMAKE_INSTALL_RPATH} lib/ ../lib/ bin/lib)
 
-set(CMAKE_CXX_STANDARD 17)
-set(CMAKE_CXX_STANDARD_REQUIRED ON)
-set(CMAKE_POSITION_INDEPENDENT_CODE ON)
+if(NOT MSVC)
+  # Compilation must be done with c++17 for NUClear to work
+  set(CMAKE_CXX_STANDARD 17)
+  set(CMAKE_CXX_STANDARD_REQUIRED ON)
+  set(CMAKE_POSITION_INDEPENDENT_CODE ON)
+endif()
```

Comment:
This should come in from master

Reply:
It did, you're commenting on an outdated diff